src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
+        if (perf == null) {
+            perf = new Perfection();
+        }
+        return perf;
+    }
```

Comment:
the idea here is to get a singleton? if so, use a private constructor.
also, what's the purpose of this method? all the rest of the methods are static. you get an instance from this class, but it has no behavior